site/pages/create-consequence-stops.page.tsx

```diff
@@ -1,7 +1,9 @@
+import { uniqueId } from "lodash";
```

Comment:
When importing from lodash we should use the modular approach to avoid importing the whole library (since it's quite hefty)

```suggestion
import uniqueId from "lodash/uniqueId";
```

Reply:
done